user-interface/src/data-verification/ConsolidationOrderAccordion.test.tsx

```diff
@@ -139,14 +140,20 @@ describe('ConsolidationOrderAccordion tests', () => {
 
   test('should correctly enable/disable buttons', async () => {
     renderWithProps();
-    const checkbox = screen.getByTestId(`${order.id}-case-list-checkbox-0`);
+    const checkbox: HTMLInputElement = screen.getByTestId(
+      `checkbox-${order.id}-case-list-case-selection-0`,
+    );
+
+    const includeAllButton = document.querySelector(`#${order.id}-checkbox-toggle-button`);
```

Comment:
Again -- we shouldn't be starting a DOM ID with a number, so we should prepend alpha text.

Reply:
Or place the order.id at the end of the ID

Reply:
I placed it at the end.